packages/react-components/src/components/AttachmentCard.tsx

```diff
@@ -149,16 +150,64 @@ export const _AttachmentCard = (props: _AttachmentCardProps): JSX.Element => {
                 }).iconName
               }
             />
-          </Stack>
-          <Stack className={attachmentNameContainerClassName}>
-            <Text className={attachmentNameTextClassName}>{attachmentName}</Text>
-          </Stack>
-          <Stack verticalAlign="center" className={actionIconClassName}>
-            {actionIcon && actionIcon}
-          </Stack>
-        </Stack>
-        {showProgressIndicator && <ProgressIndicator percentComplete={progress} styles={progressIndicatorStyles} />}
-      </Stack>
+          }
+          header={
+            <div className={fileNameContainerClassName}>
```

Comment:
trivial and can be skipped: we probably could apply styles to `CardHeader` instead of adding an additional div

`<CardHeader
        className={styles.header}.../>`

Reply:
Spent a bit of time, couldn't get it to work easily, will leave it as is for now